files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -550,16 +550,6 @@
     regexp: 'disk\/by-id\/wwn-'
     replace: 'disk/by-id/wwn/'
 
-#
-# Enable CRA for external variants
-#
-- command: pam-auth-update {{ item }}
-  with_items:
-  - --enable challenge-response
-  - --remove unix
-  when:
-    - variant is regex("external-.*")
```

Comment:
maybe change this to:
```
when:
 - variant != "external-minimal"
 - variant is regex("external-.*")
 ```
 
 as-is, this will disable CRA for the product, right?

Reply:
I do not intend to merge this PR. It was designed to create a one-time build and I posted the PR to make sure that we're not exposing anything of concern to anybody that uses this build.